apps/schedule.py

```diff
@@ -58,12 +58,14 @@ def _get_proposal_dict(proposal, favourites_ids):
         'source': 'database',
         'link': external_url('.line_up_proposal', proposal_id=proposal.id),
     }
-    if proposal.type == 'workshop':
+    if proposal.type in ['workshop', 'youthworkshop']:
         res['cost'] = proposal.cost
+        res['equipment'] = proposal.participant_equipment
+        res['age_range'] = proposal.age_range
```

Comment:
Could you use display_age_range and display_participant_equipment here? We allow proposers to override the final values.

Reply:
Of course! I couldn't find the correct model for `event` or `proposal`, it's of course called cfp 😆  977d3e7